scripts/publish-process/update-docs-deps.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+#
+# Backpack - Skyscanner's Design System
+#
+# Copyright 2016-2021 Skyscanner Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+BRANCH_NAME="autobump-upgrade-bpk-dependencies"
+
+echo "--- Cloning Backpack docs ---"
+git clone git@github.com:skyscanner/backpack-docs.git
+
+cd backpack-docs
+
+echo "--- Checkout upgrade branch ---"
+git checkout -b $BRANCH_NAME
+
+echo "--- Updating bpk dependencies ---"
+npx npm-check-updates "/^bpk\\-(.*)$/" -u && npm i
+
+echo "--- Commiting to GitHub ---"
+git add .
+git commit -m "Automated upgrade of bpk dependencies"
+git push --set-upstream origin $BRANCH_NAME
+
+echo "--- Create PR on GitHub ---"
+gh pr create --title "Automated bump for Backpack dependencies" --body "This is an automated PR generated as part of the publish process in Backpack"
```

Comment:
This assumes the publisher is signed in to GH for command line. Probably an acceptable assumption.

Do we need to set the title? By default doesn't it use the commit message?

Reply:
Yeah I guess we can exclude the title - I don't think personally it matters what we do for this and if we need to get picky about setting it or not